docs/tutorial.rst

```diff
@@ -1,10 +1,12 @@
-Tutorial
-^^^^^^^^
-
 This section covers the fundamentals of developing with *librosa*, including
 a package overview, basic and advanced usage, and integration with the *scikit-learn*
 package.  We will assume basic familiarity with Python and NumPy/SciPy.
 
+.. toctree::
+   :maxdepth: 2
+
+   auto_tutorials/index
+
 
```

Comment:
Note: the plan is to remove the remainder of this file once the intro section is complete.